code/datums/martial/security.dm

```diff
@@ -0,0 +1,118 @@
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated.</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if (H.mind.martial_art.streak == "forceful_disarm")
+		owner.visible_message("<span class='danger'>[owner] assumes a neutral stance.</span>", "<b><i>Your next attack is cleared.</i></b>")
+		H.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] assumes the Forceful Disarm stance!</span>", "<b><i>Your next attack will be a Forceful Disarm.</i></b>")
+		H.mind.martial_art.streak = "forceful_disarm"
+
+/datum/action/pressure_point_strike
+	name = "Pressure Point Strike - Deals stamina damage to a targeted limb"
+	icon_icon = 'icons/mob/actions/actions_items.dmi'
+	button_icon_state = "PressurePoint"
+
+/datum/action/pressure_point_strike/Trigger()
+	if(owner.incapacitated())
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated.</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if (H.mind.martial_art.streak == "pressure_point_strike")
+		owner.visible_message("<span class='danger'>[owner] assumes a neutral stance.</span>", "<b><i>Your next attack is cleared.</i></b>")
+		H.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] assumes the Pressure Point stance!</span>", "<b><i>Your next attack will be a Pressure Point Strike.</i></b>")
+		H.mind.martial_art.streak = "pressure_point_strike"
+
+
+/datum/martial_art/security_cqc/teach(mob/living/carbon/human/H, make_temporary=0)
+	if(..())
+		to_chat(H, "<span class = 'userdanger'>You know the arts of [name]!</span>")
+		to_chat(H, "<span class = 'danger'>Place your cursor over a move at the top of the screen to see what it does.</span>")
+		forcefuldisarm.Grant(H)
+		pressurepointstrike.Grant(H)
+
+/datum/martial_art/security_cqc/on_remove(mob/living/carbon/human/H)
+	to_chat(H, "<span class = 'userdanger'>You suddenly forget the arts of [name]...</span>")
+	forcefuldisarm.Remove(H)
+	pressurepointstrike.Remove(H)
+
+/datum/martial_art/security_cqc/proc/check_streak(mob/living/carbon/human/U, mob/living/carbon/human/T)
+	switch(streak)
+		if("forceful_disarm")
+			forceful_disarm(U, T)
+			streak = ""
+			return 1
+		if("pressure_point_strike")
+			pressure_point_strike(U, T)
+			streak = ""
+			return 1
+	return 0
+
+/datum/martial_art/security_cqc/proc/forceful_disarm(mob/living/carbon/human/U, mob/living/carbon/human/T)//User is U, target is T
+	T.visible_message("<span class = 'warning'>[U] forcefully disarms [T]!</span>", "<span class = 'userdanger'>[U] forcefully disarms you!", null, COMBAT_MESSAGE_RANGE)
+	playsound(get_turf(U), 'sound/effects/grillehit.ogg', 50, 1, -1)
+	var/obj/item/activeItem = T.get_active_held_item()
+	T.dropItemToGround(activeItem)
+	var/disarmDir = get_dir(U, T)
+	var/turf/throwAt = get_ranged_target_turf(activeItem, disarmDir, 2)
+	activeItem.throw_at(throwAt, 7, 1)
+	log_combat(U, T, "Forceful Disarm")
```

Comment:
Doesnt mean you should add more.

Reply:
I am still so confused by what this means

Reply:
Oh I am dumb this is part of the above